contracts/strategies/aave/strategy-aave-dai.sol

```diff
@@ -13,7 +13,7 @@ import "../../interfaces/aave.sol";
 import "../strategy-base.sol";
 import "../strategy-joe-farm-base.sol";
 
-contract StrategyAaveDai is StrategyBase, Exponential {
+contract StrategyAaveDaiE is StrategyBase, Exponential {
```

Comment:
for the `aave-daie` snowglobe, a new file was introduced with an 'e' suffix. 

Is this a renaming of AaveDai to AaveDaiE snowglobe and strategy or new ones?  does it make sense to keep `contracts/snowglobes/aave/snowglobe-aave-dai.sol` ?

Reply:
If we're renaming the `aave-dai`contract to `aave-daie` and deprecating the `aave-dai` pool, we should remove  the old snowglobe as a new one (aave-daie) is included in this branch